web/admin.py

```
@@ -770,3 +772,23 @@ class MembershipSubscriptionEventAdmin(admin.ModelAdmin):
     raw_id_fields = ("user", "membership")
     readonly_fields = ("created_at",)
+
+
+@admin.register(VoiceChatRoom)
+class VoiceChatRoomAdmin(admin.ModelAdmin):
+    list_display = ("name", "created_by", "created_at", "is_active", "participant_count")
+    list_filter = ("is_active", "created_at")
+    search_fields = ("name", "created_by__username")
+    date_hierarchy = "created_at"
+
+    def participant_count(self, obj):
```

Ruff (0.8.2)

Missing return type annotation for public function `participant_count`

(ANN201)